debian/control

```diff
@@ -0,0 +1,37 @@
+Source: ultrastardx
+Section: games
+Priority: optional
+Maintainer: Markus Meier <memcopy@epic-bytes.net>
+Build-Depends: debhelper (>= 10),
+ autotools-dev,
+ fpc (>= 3.0.0),
+ libpng-dev,
+ libfreetype6-dev,
+ libsqlite3-dev,
+ libsdl2-dev,
+ libsdl2-image-dev,
+ liblua5.3-dev,
+ libavcodec-dev,
+ libavformat-dev,
+ libavutil-dev,
+ portaudio19-dev,
+ libswscale-dev,
+ libprojectm-dev,
+ libpcre3-dev
```

Comment:
Any reason why you don't want to build with support for portmidi?

Reply:
I didn't notice that USDX has any midi support. What is it used for? Can it handle midi files?

Reply:
USDX contains an editor which allows importing notes and lyrics from midi files.

Reply:
Ok i will also add a dependency to libportmidi-dev.